app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss

```diff
@@ -1,4 +1,4 @@
-$dropdown-height: 373px; // Allows for entire hero_simple text to display from $screen-md and up
+$dropdown-height: 345px; // Allows for entire hero_simple text to display from $screen-md and up
```

Comment:
This falls down if the user is an admin & ambassador. In the future we definitely want to handle these dropdown heights differently but for now, I think it's safest to just be more generous w/ the prescribed height.

![screen shot 2017-01-04 at 1 29 45 pm](https://cloud.githubusercontent.com/assets/3814998/21654042/e4a7c904-d281-11e6-8be6-d3fff82af0f5.png)